src/fiatconnect/ReviewScreenWrapper.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect } from 'react'
+import { Screens } from 'src/navigator/Screens'
+import { StackScreenProps } from '@react-navigation/stack'
+import { StackParamList } from 'src/navigator/types'
+import { useSelector, useDispatch } from 'react-redux'
+import { StyleSheet, ActivityIndicator, View } from 'react-native'
+import colors from 'src/styles/colors'
+import {
+  fiatConnectQuotesErrorSelector,
+  cachedQuoteParamsSelector,
+} from 'src/fiatconnect/selectors'
+import variables from 'src/styles/variables'
+import { refetchQuote } from 'src/fiatconnect/slice'
+import { navigate } from 'src/navigator/NavigationService'
+
+type Props = StackScreenProps<StackParamList, Screens.FiatConnectReviewWrapper>
+
+export default function FiatConnectReviewScreenWrapper({ route }: Props) {
+  const { providerId, kycSchema } = route.params
+  const dispatch = useDispatch()
+  const fiatConnectQuotesError = useSelector(fiatConnectQuotesErrorSelector)
+  const cachedQuoteParamsList = useSelector(cachedQuoteParamsSelector)
+
+  useEffect(() => {
+    const cachedQuoteParams = cachedQuoteParamsList?.[providerId]?.[kycSchema]
```

Comment:
Should we also check balances before fetching a quote? If in case the cached amount is no longer available?

Reply:
We decided on not performing this check for now since it isn't strictly necessary; if a user proceeds to attempt to finalize the transfer, it will just fail and they'll receive an error message with an option to try again.

Reply:
^this was discussed in a pre refinement meeting on 10/13